crates/bevy_log/src/once.rs

```diff
@@ -0,0 +1,99 @@
+/// Call `trace!(...)` once per call site.
```

Comment:
Nit: Could you make `trace!(...)` a doc link to the `trace!` macro? Same for the other macros.